styles/modules/_userPage.scss

```diff
@@ -78,5 +60,152 @@
         box-sizing: border-box;
       }
     }
+
+    .userPageStore {
+      .userPageSearchBar {
+        margin-bottom: $padMd;
+        height: $barSm;
+
+        .ctrl,
+        .btn {
+          height: $barSm;
+          line-height: $barSm;
+        }
+
+        .btnLayoutFullWidth {
+          display: none;
+        }
+      }      
+
+      .storeFilters {
+        flex-basis: 228px;
+        margin-right: $padMd;
+
+        & > * {
+          margin-bottom: $pad;
+
+          &:last-child {
+            margin-bottom: 0;
+          }
+        }
+
+        .categoryFilter {
+          max-height: 250px;
+          overflow: auto;
+
+          .moreCatsWrap {
+            .moreCats, .showLess {
+              display: none;
+            }
+
+            &.expanded {
+              .moreCats, .showLess {
+                display: block;
+              }
+
+              .showMore {
+                display: none;
+              }
+            }
+          }
+        }
+      }
+
+      .storeListings {
+        flex-shrink: 1;
+        width: 100%;
```

Comment:
Instead of flex-shrink 1 and width" 100%, you want this to be flex-grow 1, that will make it grow to fill the remaining space.